inspect

```diff
@@ -0,0 +1,127 @@
+import os
+import json
+from os import listdir
+import os
+
+def inspectSummary(tempdir, newName):
+    os.chdir(os.path.join(tempdir + '/' + newName))
+
+    openRead = open('SUMMARY', 'r')
+    wantedValues = ['BOARD', 'FIRMUX', 'BUILD', 'LOAD', 'RAM', 'print', 'UPTIME']
+    summaryFileContent = []
+
+    for line in openRead:
+        for value in wantedValues:
+
+            if value in line:
+                summaryFileContent.append(line)   
+
+    dict = {}
+
+    size = len(summaryFileContent)
+    for i in range(size):
+
+        partitionedString = summaryFileContent[i].partition('=')
+
+        dict[partitionedString[0]] = partitionedString[2]
+        if partitionedString[0] == 'LOAD':
+            dict['CPU LOAD'] = dict.pop('LOAD')
+
+    for j in dict:
+        if j == 'RAM':
+            ramKB =  dict[j].partition(' ')
+        if j == 'CPU LOAD':
+            cpuLoad = dict[j].split(' ')
+
+    with open("SUMMARY", "r") as f:
+        content = f.read()
+
+    return dict, ramKB, cpuLoad, content
+
+
```

Comment:
Dvigubi tarpai kode yra visiškai nereikalingi, jie tik išpučia faila be jokios prasmės.